Bottom: 1eeadf5f88dbd863967db788ff43cba36657ba9c
Top:    7ef03c13f649bcf64178d7acf16a1b80b80392a3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 14:01:38 +0000

Refresh of add-some-annotations-to-help-o

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 5e5e822..4808d9b 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -869,7 +869,7 @@ M_POLYAREA_intersect (jmp_buf * e, POLYAREA * afst, POLYAREA * bfst, int add)
       for (curcA = a->contours; curcA != NULL; curcA = curcA->next)
 	if (curcA->Flags.status == ISECTED)
 	  {
-	    the_list = add_descriptors (curcA, 'A', the_list)
+	    the_list = add_descriptors (curcA, 'A', the_list);
 	    if (UNLIKELY (the_list) == NULL)
 	      error (err_no_memory);
 	  }
